rscripts/YAL5-codon-pairs.R

```diff
@@ -0,0 +1,1455 @@
+rm(list = ls())
+
+# Given an h5 file, GFF file and .tsv file, this script creates a metafeature
+# plot of a feature of interest or a table which contains the relative counts
```

Comment:
`plot of a feature of interest` -> `plot of a window of positions around the feature of interest`